### PR TITLE
Add docs on how to log into Grafana

### PR DESCRIPTION
**Pull request by SomtochiAma**
Signed-off-by: Somtochi Onyekwere <somtochionyekwere@gmail.com>

### CODE REVIEW

File: content/en/docs/guides/monitoring.md [hunk 1]
```diff
@@ -70,6 +70,14 @@ You can access Grafana using port forwarding:
 kubectl -n monitoring port-forward svc/kube-prometheus-stack-grafana 3000:80
 ```
 
+## Log into the Grafana dashbooard
```
Comment (stefanprodan) [inline content/en/docs/guides/monitoring.md:73]: No need for a new header here, please remove it.

File: content/en/docs/guides/monitoring.md [hunk 2]
```diff
@@ -70,6 +70,14 @@ You can access Grafana using port forwarding:
...
+To log into the Grafana dashboard, use the following credentials:
```
Comment (stefanprodan) [inline content/en/docs/guides/monitoring.md:74]:
```suggestion
To log into Grafana, use the following credentials:
```

File: content/en/docs/guides/monitoring.md [hunk 3]
```diff
@@ -70,6 +70,14 @@ You can access Grafana using port forwarding:
...
+```
+username: admin
+password: prom-operator
+```
+These are the default values from the [kube-prometheus helm chart](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L620)
```
Comment (stefanprodan) [inline content/en/docs/guides/monitoring.md:79]:
```suggestion
These are the default values from the [kube-prometheus-stack chart](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L620).
```

Comment (hiddeco) [inline content/en/docs/guides/monitoring.md:79]:
```suggestion
## Log in to the Grafana dashbooard

To log in to the Grafana dashboard, you can use the default credentials from the [kube-prometheus helm chart](https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L620):

```yaml
username: admin
password: prom-operator
```
```

Comment (hiddeco) [reply content/en/docs/guides/monitoring.md:79]: * "Log into" -> "log in to"
* Explain where the credentials are from in a single sentence
* Add `yaml` to example block for highlighting

**Review by stefanprodan — APPROVED** [review]
LGTM

Thanks @SomtochiAma ☕

**Pull request merged**